extensions/wikia/ArticleVideo/styles/ooyala/base/_type.scss

```diff
@@ -0,0 +1,109 @@
+@at-root {
+  @font-face {
+    font-family: "ooyala-slick-type";
+    src: url('#{$font-path}/ooyala-slick-type.eot');
+    src: url('#{$font-path}/ooyala-slick-type.eot?#iefix') format('embedded-opentype'),
+    url('#{$font-path}/ooyala-slick-type.woff') format('woff'),
+    url('#{$font-path}/ooyala-slick-type.ttf') format('truetype'),
+    url('#{$font-path}/ooyala-slick-type.svg#alice') format('svg');
+    font-weight: normal;
+    font-style: normal;
+
+  }
+  /* roboto condensed regular */
+  @font-face {
+    font-family: 'Roboto Condensed';
+    font-style: normal;
+    font-weight: 400;
+    src: local('Roboto Condensed'), local('RobotoCondensed-Regular'),
+    url('#{$font-path}/RobotoCondensed-Regular.woff2') format('woff2'),
+    url('#{$font-path}/RobotoCondensed-Regular.woff') format('woff'),
+    url('#{$font-path}/RobotoCondensed-Regular.ttf') format('truetype');
+    unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2212, U+2215, U+E0FF, U+EFFD, U+F000;
+  }
+  /* roboto condensed bold */
+  @font-face {
+    font-family: 'Roboto Condensed Bold';
+    font-style: normal;
+    font-weight: 700;
+    src: local('Roboto Condensed Bold'), local('RobotoCondensed-Bold'),
+    url('#{$font-path}/RobotoCondensed-Bold.woff2') format('woff2'),
+    url('#{$font-path}/RobotoCondensed-Bold.woff') format('woff'),
+    url('#{$font-path}/RobotoCondensed-Bold.ttf') format('truetype');
+    unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2212, U+2215, U+E0FF, U+EFFD, U+F000;
+  }
+
+  /* roboto regular */
+  @font-face {
+    font-family: 'Roboto';
```

Comment:
This font is used only for displaying the video's time. I think it's better to remove all the fonts but `ooyala-slick-type` and use Helvetica where needed.